.coveragerc

```diff
@@ -4,8 +4,8 @@ branch = True
 [report]
 show_missing = True
 omit =
-    google/cloud/iam_credentials/__init__.py
-    google/cloud/iam_credentials/gapic_version.py
+    google/cloud/iam/__init__.py
```

Comment:
Im Java iam and iamcredentials are different library. If this change is intentional, that's fine.

Reply:
Acknowledged. `iamcredentials` was shipped first as `google-cloud-iam` instead of `google-cloud-iamcredentials`. I think this is what caused us to merge the 2 APIs. When `google/iam` v2/v2beta came along we decided to add them to the existing `google-cloud-iam` package. We don't have plans to split up iam and iamcredentials. Let's continue the discussion in https://github.com/googleapis/google-cloud-python/issues/11572.